app-text/nfoview/nfoview-1.15.1.ebuild

```diff
@@ -7,18 +7,17 @@ PYTHON_COMPAT=( python3_{4,5,6} )
 
 inherit distutils-r1 fdo-mime gnome2-utils
 if [[ ${PV} == "9999" ]] ; then
-	EGIT_REPO_URI="git://github.com/otsaloma/nfoview.git
-		https://github.com/otsaloma/nfoview.git"
+	EGIT_REPO_URI="https://github.com/otsaloma/${PN}.git"
 	inherit git-2
 	SRC_URI=""
 	#KEYWORDS=""
 else
-	SRC_URI="http://download.gna.org/nfoview/${PV:0:4}/${P}.tar.xz"
-	KEYWORDS="amd64 x86"
+	SRC_URI="https://github.com/otsaloma/${PN}/archive/${PV}.tar.gz"
```

Comment:
This results in a very bad filename. Use `->` to rename it to something sane. Though I'm not sure if it's really meaningful to update it when the old file is kept on mirrors. Maybe just add the new URI in a comment for whoever's going to bump it in the future.